internal/dkg/actions_active.go

```diff
@@ -36,6 +36,75 @@ func (d *Process) StartNetwork(ctx context.Context, options *drand.FirstProposal
 		return nil, err
 	}
 
+	d.log.Infow("running DKG command", "command", commandName, "beaconID", beaconID)
+	// map the current state to the latest state, depending on the type of packet
+	var packetToGossip *drand.GossipPacket
+	var afterState *DBState
+	switch c := command.Command.(type) {
```

Comment:
any chance command might be `nil`? (e.g. assuming attacker tries to crash nodes)